sage-ios/SAGE/SAGE/LoginController.swift

```diff
@@ -35,35 +37,67 @@ class LoginController: UIViewController, UITextFieldDelegate {
         })
     }
     
-    func textFieldShouldReturn(textField: UITextField) -> Bool {
+    //
+    // MARK: - Methods to handle navigation
+    //
+    
+    func pushRootTabBarController() {
+        LoginHelper.setUserSingleton()
+        let rootTabBarController = RootTabBarController()
+        self.presentViewController(rootTabBarController, animated: false, completion: nil)
+    }
+    
+    func showError(message: String) {
```

Comment:
also make the errorview a property of the view controller, and make sure it doesnt keep on reshowing evertime i try to scroll.

For the orange screen. make sure to show the right error message (password too short, need berkleey.edu address

Can you actaully also make this a completely opaque UIColor.lightredColor with a shadow on the bottom?